File: legishub-service/wallet-service/src/main/java/br/com/cin/aps/apjcardservice/service/card/CardBrandService.java
package br.com.cin.aps.apjcardservice.service.card;

import br.com.cin.aps.apjcardservice.entity.Cartao;
import br.com.cin.aps.apjcardservice.dto.CardDTO;

public interface CardBrandService {

    boolean sendChargeRequest(Cartao cartao, Double valor) ;

    boolean sendValidationRequest(CardDTO cartao);
}

File: legishub-service/apj-service/src/main/kotlin/br/com/cin/aps/apjservice/client/CardBrandService.kt
package br.com.cin.aps.apjservice.client;

import br.com.cin.aps.apjservice.entity.Cartao
import org.springframework.stereotype.Service

@Service
interface CardBrandService {
     fun validar(cartao : Cartao) : Boolean
     fun debitar(cartao : Cartao, valor : Int) : Int
}
